tests/integration/run_tests.py

```diff
@@ -139,7 +145,7 @@ def run_tests(args, tests):
         n_to_run = 0
         selective = True
         for test_id, test in enumerate(tests.values()):
-            if len(test) == 3 and test[2] == "local":
+            if len(test) >= 3 and test[2] == "local":
                 args.ids.append(test_id + 1)
```

Comment:
Why the need to add the +1 offest to the ids here?

Reply:
Please see my comment above. It is to accommodate for the increase in the tuple size.